prov/net/src/xnet_domain.c

```diff
@@ -141,6 +163,10 @@ static int xnet_domain_close(fid_t fid)
 	domain = container_of(fid, struct xnet_domain,
 			      util_domain.domain_fid.fid);
 
+	ret = xnet_del_wait_eq_list(domain);
```

Comment:
I guess it would align with the open path to wrap this call
```
if (!fabric->progress.auto_progress) {
    ret = xnet_del_wait_eq_list(...)
}
```
Though I think it's harmless to always call it.  If the progress epoll fd isn't attached to the wait epoll fd, the underlying code handles it.

Reply:
Fixed. Thanks!